analyses/molecular-subtyping-embryonal/README.md

```diff
@@ -1,10 +1,80 @@
 # Molecular Subtyping non-MB, non-ATRT Embryonal Tumors
 
+**Module authors:** Stephanie J. Spielman([@sjspielman](https://github.com/sjspielman)) and Jaclyn Taroni ([@jaclyn-taroni](https://github.com/jaclyn-taroni))
+
 **Note: The files in the `subset-files` directory were generated via `02-generate-subset-files.R` using the the files in the [version 13 data release](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/444).**
 
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Usage](#usage)
+- [Folder Content](#folder-content)
+  - [Define samples for subsetting](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/01-samples-to-subset.nb.html)
+  - [Generate subset files](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-embryonal/02-generate-subset-files.R)
+  - [Clean C19MC miRNA cluster data](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/03-clean-c19mc-data.nb.html)
+  - [Construct summary tables](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/04-table-prep.nb.html)
+- [Folder Structure](#folder-structure)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Usage
+
 When re-running this module, you may want to regenerate the subset files using the most recent data release.
-Files will be regenerated using the symlinked files in `data` by default when running:
+Files will be regenerated using the symlinked files in `data` by default when running from the command line as follows:
 
-```sh
+```
 bash run-embryonal-subtyping.sh
 ```
+
+
+## Folder Content
+
+[`01-samples-to-subset.Rmd`](https://alexslemonade.github.io/OpenPBTA-analysis/analyses/molecular-subtyping-embryonal/01-samples-to-subset.nb.html) is a notebook written to identify samples to include in subset files for the purpose of molecularly subtyping non-MB and non-ATRT embryonal tumors.
+The samples are identified using the following criteria:
+
+1. An RNA-seq biospecimen sample includes a _TTYH1_ fusion (5' partner).
```

Comment:
Let's add a link to this comment here: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/401#issuecomment-573669727